packages/core/src/types/VolumeProps.ts

```diff
@@ -31,10 +36,19 @@ interface VolumeProps {
   /** Image data representing the volume */
   imageData?: vtkImageData;
 
-  /** Scalar data representing the volume's intensity values */
-  scalarData: PixelDataTypedArray | Array<PixelDataTypedArray>;
+  /**
+   * The new volume model which solely relies on the separate image data
+   * and do not cache the volume data at all
+   */
+  voxelManager?: VoxelManager<number> | VoxelManager<RGB>;
```

Comment:
Should the type here be exported as it's own type - so that if we add something in the future like RGBF with F being fluorescence, that we can use it directly?  I don't know if that seems likely, just concerned it might occur.

Reply:
I would recommend doing it when it happens